RELEASE/scripts/autoscend.ash

```diff
@@ -2156,7 +2156,7 @@ boolean doBedtime()
 			done = true;
 		}
 	}
-	if(!done)
+	if(!done && !in_zelda())
```

Comment:
1. Can drink is a mafia function we cannot change
https://wiki.kolmafia.us/index.php?title=Can_drink
2. wiki page for it was wrong, it also returns false in plumber
3. making it return false will skip the rest of the doBedtime() function which is what we are trying to prevent.

What I could do is modify
```
boolean done = (my_inebriety() > inebriety_limit()) || (my_inebriety() == inebriety_limit() && my_familiar() == $familiar[Stooper]);
```
to add to it a test for can drink instead of a zelda test. This would incidentally resolve the exact same bug in teetotaler and oxygenerian paths as well. So I should do that.

Reply:
Also, change my_adventure() checks to what?

Reply:
correction applied commit 943fbb2